### ISSUE TITLE
lonboard v0.4.0  polyogn visualization bug

### ISSUE THREAD

**Issue by giswqs** [developmentseed/lonboard#241]
I tried to render a polygon. GeoPandas can render it properly. However, lonboard seems getting some weird edge rendering issue. Not sure if this a lonboard issue or the deck.gl JS library issue. 

```python
import geopandas as gpd
from lonboard import viz

# Downloaded from 'https://data.source.coop/giswqs/nwi/state_boundary/AR.parquet'
gdf = gpd.read_parquet('AR.parquet')
gdf.explore()
```
![image](https://github.com/developmentseed/lonboard/assets/5016453/5a7d80f6-9b44-49c4-9ba8-eb13e0e1d0c3)

```python
viz(gdf, get_fill_color=[0, 0, 255, 128])
```
![image](https://github.com/developmentseed/lonboard/assets/5016453/aac530f8-e005-483a-9af6-4b3fbd80c36b)

**Comment by giswqs** [comment]
Another example. 
https://data.source.coop/giswqs/nwi/wetlands/DC_Wetlands.parquet
![image](https://github.com/developmentseed/lonboard/assets/5016453/84ece467-7227-4b41-a610-4875eb813d13)

![image](https://github.com/developmentseed/lonboard/assets/5016453/52eb6f6d-12d5-40f2-8b4d-14c9bd23be07)

**Comment by giswqs** [comment]
It seems this is a new bug introduced in v0.4.0. Downgrading to v0.3.0 sloves the issue. 
![image](https://github.com/developmentseed/lonboard/assets/5016453/ff74ca4d-5b31-4921-baaa-76fe7a5b7ed3)

**Comment by kylebarron** [comment]
Thanks for the reproducible examples. This should be fixed in 0.4.1, being published now
